pyproject.toml

```diff
@@ -54,8 +54,11 @@ visualize = [
 devtools = [
     "tqdm"
 ]
+groundwater_illustration_matching = [
```

Comment:
It still doesn't quite work for me to do install scikit-image with `pip install -e '.[all]'` .
Probably need to use hyphens here instead of underscores (i.e. `groundwater-illustration-matching`).
I think there's some normalisation happening in the background that interferes with underscores: see e.g. https://peps.python.org/pep-0685/
We had the same issue already once before with Renato: https://github.com/swisstopo/swissgeol-boreholes-dataextraction/pull/54#discussion_r1625849873

Reply:
Ah thank you for letting me know!